src/main/java/org/demo/service/mail/MailSendingService.java

```diff
@@ -106,4 +121,21 @@ private SocketNotificationDTO getSocketNotificationDTOWithError(SocketNotificati
 				.build();
 	}
 
+	private List<NotificationLinkDTO> getLinks(String link, Optional<Meeting> meeting) {
+		List<NotificationLinkDTO> result = new ArrayList<>();
+		result.add(NotificationLinkDTO.builder()
+				.drillDownType(DrillDownType.EXTERNAL_NEW.getValue())
+				.drillDownLabel(link)
+				.drillDownLink(HTTP + link)
+				.build());
+
+		meeting.ifPresent(value -> result.add(NotificationLinkDTO.builder()
+				.drillDownType(DrillDownType.INNER.getValue())
+				.drillDownLabel(String.format("Meeting %s", value.getId()))
+				.drillDownLink(String.format("/screen/meeting/view/meetingview/meeting/%s", value.getId()))
```

Comment:
just concatenate in link. dont use string format - it is hard to read for complex urls